aiokafka/client.py

```diff
@@ -587,6 +588,7 @@ def _check_api_version_response(self, response):
             # ((2, 6, 0), DescribeClientQuotasRequest[0]),
             ((2, 5, 0), DescribeAclsRequest_v2),
             ((2, 4, 0), ProduceRequest[8]),
+            ((2, 3, 0), SyncGroupRequest[3]),
```

Comment:
This list is used to determine broker version. One feature is enough for each version, so this line is excessive.